src/main/java/org/sagebionetworks/web/client/presenter/ProfilePresenter.java

```diff
@@ -334,14 +358,43 @@ public void initializeShowHideVerification(boolean isOwner) {
 			view.setVerificationAlertVisible(isVerificationAlertVisible);
 			//show the submit verification button if the full alert isn't visible
 			view.setVerificationButtonVisible(!isVerificationAlertVisible);
-		} else {
-			//not the owner
-			//hide message
-			view.setVerificationAlertVisible(false);
-			view.setVerificationButtonVisible(false);
 		}
 	}
 	
+	public void getIsACTMemberAndShowVerificationUI(final VerificationSubmission submission) {
+		if (authenticationController.isLoggedIn()) {

```

Comment:
It should not be asking this question here, it should be asked once (when user session data is acquired).